.gitleaks.toml

```diff
@@ -0,0 +1,4 @@
+[allowlist]
+  description = "Documentation contains example secrets and passwords"
+  paths = [
+    "docs/docsite/rst/administration/oauth2_token_auth.rst"
```

Comment:
Is it not a problem that this doesn't have list closure `]`?

Reply:
gosh, how silly of me. fixed. and thanks for spotting this.